### DISCUSSION TITLE
Add judgement time to beatmap attribute text

### DISCUSSION THREAD

**Discussion by developomp** [ppy/osu#22300] [Ideas]
It's a simple type of beatmap attribute text that calculates the ± time window in millisecond based on the OD. For example, in osu!std with OD10 NM the values will be `19.5` for 300, `59.5` for 100, etc. This can be put next to the error meter bar or the newly added [judgement counter](https://github.com/ppy/osu/pull/21620).

**Comment by mk56-spn** [comment]
This shouldn't be done for now IMO ( even if wanted ) because currently DT does not show it's actual OD values in menus and such .and as such anything other than NM or non OD affecting mods  ( because the beatmap info text doesn't use any mod info yet ) will be incorrect , leading to unnecessary confusikn
